packages/react/scroll-area/src/ScrollArea.tsx

```diff
@@ -466,7 +467,8 @@ const ScrollAreaImpl = forwardRefWithAs<typeof ROOT_DEFAULT_TAG, ScrollAreaImplP
         <ScrollAreaRefsContext.Provider value={refs}>
           <ScrollAreaContext.Provider value={context}>
             <ScrollAreaStateContext.Provider value={reducerState}>
-              <Comp
+              <Primitive
+                selector={getSelector(ROOT_NAME)}
                 {...domProps}
                 ref={ref}
```

Comment:
nit: ref after spread

Reply:
this one is already but will do the others :+1:

Reply:
haha 🤦‍♂️ got too trigger happy there 😀